libs/json/src/test/test_json_simple.c

```diff
@@ -24,6 +24,7 @@
 #include "json/json.h"
 
 static char *output = "{\"KeyBool\": true,\"KeyInt\": -1234,\"KeyUint\": 1353214,\"KeyString\": \"foobar\",\"KeyStringN\": \"foobarlong\",\"KeyIntArr\": [153,2532,-322]}";
+static char *output2 = "{\"KeySructArr\" : [{true, 2003, 'foobar'},{false, 2035, 'foobarst'}]}";
```

Comment:
I assume you meant this to be:
static char *output2 = "{\"KeySructArr\" : [{\"KeyBool\":true, \"KeyInt\":2003, \"KeyString\":\"foobar\"},"
           "{\"KeyBool\":false, \"KeyInt\":2035, \"KeyString\":\"foobarst\"}]}";

I think you need the key to go with every value. https://www.ietf.org/rfc/rfc4627.txt